gitlabform/gitlab/tags.py

```diff
@@ -14,3 +14,16 @@ def create_tag(self, project_and_group_name, tag_name, ref, message=None):
         }
         return self._make_requests_to_api("projects/%s/repository/tags", project_and_group_name, method='POST', data=data,
                                           expected_codes=201)
+
+    def protect_tag(self, project_and_group_name, tag_name, create_access_level):
+        data = {
+            'name': tag_name
+        }
+        if create_access_level:
```

Comment:
I think this is the reason for this bug:
- https://github.com/egnyte/gitlabform/issues/172